packages/checkbox/src/FzCheckboxGroup.vue

```diff
@@ -26,11 +26,15 @@
 
 <script setup lang="ts">
 import { computed } from "vue";
-import { FzCheckboxGroupProps, ParentCheckbox } from "./types";
+import { FzCheckboxGroupProps } from "./types";
 import FzCheckboxErrorText from "./components/FzCheckboxErrorText.vue";
 import { mapSizeToClasses } from "./common";
 import FzCheckboxGroupOption from "./components/FzCheckboxGroupOption.vue";
 
+FzCheckboxGroupOption.compatConfig = {
+  MODE:3
+}
+
```

Comment:
can't this be put in the app and not here?

Reply:
No because FzCheckboxGroupOption is not exposed, therefor I cannot set the mode:3 in the app